docs/contributing.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 10
+title: "🍻 Contributing"
+---
+
+# 🍻 Contributing
+
+We welcome all contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas.
+
+## Getting Started
+The Chroma codebase is hosted on [Github](https://github.com/chroma-core/chroma) and issues are tracked on [Github Issues](https://github.com/chroma-core/chroma/issues). Please report any issues you find there making sure to fill out the correct [form for the type of issue you are reporting](https://github.com/chroma-core/chroma/issues/new/choose). In order to run Chroma locally you can follow the [Development Instructions](https://github.com/chroma-core/chroma/blob/main/DEVELOP.md).
```

Comment:
maybe this would be better as bullets? more digestable and each sentence is pretty standalone

Reply:
copy